mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -119,69 +146,51 @@ View a report of the experiment in HTML or text formats as follows.
-        -----------------------------|-----
-           Number of completed tasks |4
-        -----------------------------|-----
-
-
-
-        -----------------------------|-----
-                                 SLOs|
-        -----------------------------|-----
-                  built-in/error-rate|true
-        -----------------------------|-----
-                built-in/p95.0 (msec)|true
-        -----------------------------|-----
-
-
-          -----------------------------|-----
-                                Metrics|
-          -----------------------------|-----
-                   built-in/error-count|0
-          -----------------------------|-----
-                    built-in/error-rate|0
-          -----------------------------|-----
-                built-in/latency (msec)|3
-          -----------------------------|-----
-            built-in/max-latency (msec)|213.21
-          -----------------------------|-----
-           built-in/mean-latency (msec)|17.47
-          -----------------------------|-----
-            built-in/min-latency (msec)|4.30
-          -----------------------------|-----
-                  built-in/p50.0 (msec)|10.80
-          -----------------------------|-----
-                  built-in/p75.0 (msec)|12.40
-          -----------------------------|-----
-                  built-in/p90.0 (msec)|13.60
-          -----------------------------|-----
-                  built-in/p95.0 (msec)|14
-          -----------------------------|-----
-                  built-in/p99.0 (msec)|209.91
-          -----------------------------|-----
-                  built-in/p99.9 (msec)|212.88
-          -----------------------------|-----
-                 built-in/request-count|100
-          -----------------------------|-----
-         built-in/stddev-latency (msec)|39.90
-          -----------------------------|-----
+    ??? note "The text report looks as follows."
+        ```shell
+        Experiment summary:
+        *******************
+
+          Experiment completed: true
+          No failed tasks: true
+          Total number of tasks: 2
+          Number of completed tasks: 2
+
+        Whether or not service level objectives (SLOs) are satisfied:
+        *************************************************************
+
+          SLO Conditions                           |Satisfied
+          --------------                           |---------
+          built-in/http-latency-mean (msec) <= 100 |true
+          built-in/http-latency-p95 (msec) <= 150  |true
+          
+
+        Latest observed values for metrics:
+        ***********************************
+
+          Metric                              |value
+          -------                             |-----
+          built-in/http-error-count           |0.00
+          built-in/http-error-rate            |0.00
+          built-in/http-latency-max (msec)    |186.56
+          built-in/http-latency-mean (msec)   |11.25
```

Comment:
I don't see changes to report.go that would modify the output in this way. Is that an oversight?

Reply:
The correct template was already merged in the previous PR. This is just an update to the documentation.